generator/internal/genclient/parser/protobuf/protobuf.go

```diff
@@ -226,27 +228,29 @@ func MakeAPI(serviceConfig *serviceconfig.Service, req *pluginpb.CodeGeneratorRe
 
 	// Then we need to add the messages, enums and services to the list of
 	// elements to be generated.
-	for _, f := range req.GetSourceFileDescriptors() {
+	for _, f := range append(req.GetSourceFileDescriptors(), mixins...) {
 		var fileServices []*genclient.Service
 		fFQN := "." + f.GetPackage()
 
 		// Messages
-		for _, m := range f.MessageType {
-			mFQN := fFQN + "." + m.GetName()
-			if msg, ok := state.MessageByID[mFQN]; ok {
-				api.Messages = append(api.Messages, msg)
-			} else {
-				slog.Warn("missing message in symbol table", "message", mFQN)
+		if !isMixinPackage(f.GetPackage()) {
```

Comment:
I would have said 

```go
  if isMixinPackage(f.GetPackage()) {
      continue;
  }
```

Reply:
There is more logic todo if this is not the case.